virtualcluster/pkg/syncer/resources/configmap/checker_test.go

```diff
@@ -27,11 +27,15 @@ import (
 	core "k8s.io/client-go/testing"
 
 	"sigs.k8s.io/cluster-api-provider-nested/virtualcluster/pkg/apis/tenancy/v1alpha1"
+	"sigs.k8s.io/cluster-api-provider-nested/virtualcluster/pkg/syncer/constants"
 	"sigs.k8s.io/cluster-api-provider-nested/virtualcluster/pkg/syncer/conversion"
+	"sigs.k8s.io/cluster-api-provider-nested/virtualcluster/pkg/syncer/util/featuregate"
 	util "sigs.k8s.io/cluster-api-provider-nested/virtualcluster/pkg/syncer/util/test"
 )
 
 func TestConfigMapPatrol(t *testing.T) {
+	featuregate.DefaultFeatureGate.Set(featuregate.RootCACertConfigMapSupport, true)
```

Comment:
IIRC, we have SetFeatureGateDuringTest https://github.com/kubernetes-sigs/cluster-api-provider-nested/blob/dbcdf7c4b3825340aef3c6bf343fdb0d5cd9e8d0/virtualcluster/pkg/syncer/util/test/featuregate.go#L31-L31

Reply:
Oh nice!